### PR TITLE
ROECCT-244- Error created completing Update statements if a pre-registration RBO is added

### PR DESCRIPTION
**Pull request by krenjith-ch**
### JIRA link

https://companieshouse.atlassian.net/browse/ROECCT-244

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by ConnorOS-ch — COMMENTED** [review]
Have you checked all the uses for this function and confirmed that there aren't any cases where you would still need to return pre-reg bo's as well?

File: src/utils/application.data.ts [hunk 1]
```diff
@@ -57,15 +57,15 @@ export const mapDataObjectToFields = (data: any, htmlFields: string[], dataModel
 
 export const allBeneficialOwners = (appData: ApplicationData): Array<BeneficialOwnerIndividual | BeneficialOwnerCorporate | BeneficialOwnerGov> => {
   if (!appData.update?.no_change) {
-    return (appData.beneficial_owners_individual ?? [])
+    return (appData.beneficial_owners_individual?.filter(relevant => !relevant.relevant_period) ?? [])
```
[the remark this answers is not in the document]
Comment (krenjith-ch) [reply src/utils/application.data.ts]: updated var name

**Review by ConnorOS-ch — APPROVED** [review]
Looks good to me 👍

**Comment by krenjith-ch** [comment]
> Have you checked all the uses for this function and confirmed that there aren't any cases where you would still need to return pre-reg bo's as well?

As discussed, it is confirmed that the function is used only for update statement page

**Pull request merged**